chia/full_node/subscriptions.py

```diff
@@ -208,3 +212,31 @@ def coin_subscription_count(self) -> int:
 
     def puzzle_subscription_count(self) -> int:
         return self._puzzle_subscriptions.total_count()
+
+
+def peers_for_spend_bundle(
+    peer_subscriptions: PeerSubscriptions, conds: SpendBundleConditions, hints_for_removals: Set[bytes32]
```

Comment:
I think this function should have a unit test. A doc-comment would be nice as well